practicesharp/Numeric_Base.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    public interface Numeric_Base
+    {
+        private static string field;
+        public bool Validate(string number);
+        public double To_Decimal(string number);
+        public void print(double number);
+    }
+
+
+    class Binary : Numeric_Base
+    {
+        private static string field = "01";
+        private short weight = 2;
+        public bool Validate(string number)
+        {
+            foreach (char letter in number)
+                if (!field.Contains(letter)) return false;
+            return true;
+        }
+        public double To_Decimal(string number)
```

Comment:
This function does not convert the number to decimal, it just parses it into a double.
A better name will be "Parse".